cmd/sniproxy/sniproxy.go

```diff
@@ -68,6 +95,34 @@ func main() {
 		go s.serve(ln)
 	}
 
+	for _, forwStr := range strings.Split(*forwards, ",") {
+		if forwStr == "" {
+			break
```

Comment:
continue? (I suspect this is here to handle `*forwards == ""`, otherwise I'd suggest it should be an error)

Reply:
Done